src/CarbonAware.CLI/src/CarbonAwareCLI.cs

```diff
@@ -70,9 +70,14 @@ public async Task<IEnumerable<EmissionsData>> GetEmissions()
 
     private async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(Dictionary<string, object> props)
     {
-        IEnumerable<EmissionsData> e = await _aggregator.GetEmissionsDataAsync(props);
-
-        return await _aggregator.GetEmissionsDataAsync(props);
+        if ((bool)props[CarbonAwareConstants.Best])
```

Comment:
As written, this will always be `true` [See line 66](https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/104/files#diff-353591b2f9db8ce1f6ca2bf975ce2124c21682b8c898a9f6b3c56a049fa7ed5aR66).  But I think it might be better to eliminate this function entirely and simply move this switch up into `GetEmissions()` as the `Best` property is not actually used by the aggregator.

```c#
    public async Task<IEnumerable<EmissionsData>> GetEmissions()
    {
        IEnumerable<Location> locations = _state.Locations.Select(loc => new Location(){ RegionName = loc, LocationType = LocationType.CloudProvider });
        var props = new Dictionary<string, object>() {
            { CarbonAwareConstants.MultipleLocations, locations },
            { CarbonAwareConstants.Start, _state.Time },
            { CarbonAwareConstants.End, _state.ToTime },
        };
        if (_state.Lowest)
        {
            return await _aggregator.GetBestEmissionsDataAsync(props);
        }
        else
        {
            return await _aggregator.GetEmissionsDataAsync(props);
        }
    }
```